getting-started/glossary.md

```diff
@@ -1,51 +1,93 @@
-# GLOSSARY   
+# GLOSSARY
 
-## API
-Application Programming Interface. A software intermediary that allows two applications to talk to each other.
+This page provides a brief explanation of the key terms used in Cloud File Transfer  
 
+## API
+Application Programming Interface. Refers to a set of functions and procedures that allows two applications to interact with each other programmatically in order to access the features or data of an operating system, application or other service. 
```

Comment:
"Application Programming Interface (API) refers to a set of functions and procedures that allows two applications to interact with each other programmatically in order to access the features or data of an operating system, application or other service. "